### PR TITLE
Migrate to ghcr

### PR DESCRIPTION
**Pull request by tei-k**
# Description

https://scalar-labs.atlassian.net/browse/DLT-7849

# Done

# After merge
Need to delete `DOCKERHUGB_ACCESS_TOKEN` and create `CR_PAT` in secrets.

### CODE REVIEW

File: docs/DeployScalarDLAnsible.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ This document explains how to deploy Scalar Ledger and Envoy on Kubernetes with
...
 
 ## Preparation
 
-You need to set `DOCKERHUB_USER` and `DOCKERHUB_ACCESS_TOKEN` as env or set the values directly in the `playbook-deploy-scalardl.yml` for `docker_username` and `docker_password`.
+You need to set `DOCKER_USER` and `DOCKER_ACCESS_TOKEN` as env or set the values directly in the `playbook-deploy-scalardl.yml` for `docker_username` and `docker_password`.
 
 ```console
-export DOCKERHUB_USER=<user>
-export DOCKERHUB_ACCESS_TOKEN=<token>
+export DOCKER_USER=<user>
```
Comment (feeblefakie) [inline docs/DeployScalarDLAnsible.md:17]:
```suggestion
export DOCKER_REGISTRY_USER=<user>
```

File: docs/DeployScalarDLAnsible.md [hunk 2]
```diff
@@ -6,16 +6,16 @@ This document explains how to deploy Scalar Ledger and Envoy on Kubernetes with
...
+export DOCKER_ACCESS_TOKEN=<token>
```
Comment (feeblefakie) [inline docs/DeployScalarDLAnsible.md:18]:
```suggestion
export DOCKER_REGISTRY_ACCESS_TOKEN=<token>
```

**Review by feeblefakie — COMMENTED** [review]
Left 1 comment about the naming.

**Comment by tei-k** [comment]
I've refactored the environment variable names as follows in d5c8a8a

DOCKER_REGISTRY_PASSWORD
DOCKER_REGISTRY_USER
DOCKER_REGISTRY_SERVER
CR_PAT

**Comment by feeblefakie** [comment]
@tei-k I think we agreed on `DOCKER_REGISTRY_USERNAME`.

**Comment by tei-k** [comment]
> I think we agreed on DOCKER_REGISTRY_USERNAME

Oh! Just misstake. Fixed in 39362b6

**Comment by tei-k** [comment]
@feeblefakie 

Can you do this?
>Need to delete `DOCKERHUGB_ACCESS_TOKEN` and create `CR_PAT` in secrets.

File: playbooks/playbook-deploy-scalardl.yml [hunk 3]
```diff
@@ -5,9 +5,9 @@
   become: no
   vars:
     base_local_directory: "{{ lookup('env','SCALAR_K8S_CONFIG_DIR') }}"
-    # docker_registry: "https://index.docker.io/v2/"
-    # docker_username: "{{ lookup('env','DOCKERHUB_USER') }}"
-    # docker_password: "{{ lookup('env','DOCKERHUB_ACCESS_TOKEN') }}"
+    # docker_registry: "ghcr.io"
+    # docker_username: "{{ lookup('env','DOCKER_REGISTRY_USER') }}"
```
Comment (feeblefakie) [inline playbooks/playbook-deploy-scalardl.yml:9]:
```suggestion
    # docker_username: "{{ lookup('env','DOCKER_REGISTRY_USERNAME') }}"
```

File: playbooks/roles/scalardl/defaults/main.yml [hunk 4]
```diff
@@ -2,9 +2,9 @@ base_local_directory: ../conf
 local_helm_charts_directory: ../../../../charts/stable
 local_helm_charts_values_directory: "{{ base_local_directory }}/"
 remote_directory: "/home/{{ ansible_user }}"
-docker_registry: "https://index.docker.io/v2/"
-docker_username: "{{ lookup('env','DOCKERHUB_USER') }}"
-docker_password: "{{ lookup('env','DOCKERHUB_ACCESS_TOKEN') }}"
+docker_registry: ghcr.io
+docker_username: "{{ lookup('env','DOCKER_REGISTRY_USER') }}"
```
Comment (feeblefakie) [inline playbooks/roles/scalardl/defaults/main.yml:6]:
```suggestion
docker_username: "{{ lookup('env','DOCKER_REGISTRY_USERNAME') }}"
```

**Review by feeblefakie — COMMENTED** [review]
Left 2 suggestions!

**Comment by feeblefakie** [comment]
@tei-k One of the CIs is failing. I retried it but didn't work. Can you take a look?

File: playbooks/roles/scalardl/defaults/main.yml (hunk 4 above)
Comment (tei-k) [reply playbooks/roles/scalardl/defaults/main.yml]: 😇 Fixed in 6d15bca

**Comment by tei-k** [comment]
>One of the CIs is failing. I retried it but didn't work. Can you take a look?

Maybe need to add new secret
https://github.com/scalar-labs/scalar-kubernetes/pull/103#issuecomment-755848269

**Comment by feeblefakie** [comment]
@tei-k After updating the secret, all the CIs work fine now! 🙇

**Review by feeblefakie — APPROVED** [review]
LGTM!

**Comment by feeblefakie** [comment]
@ymorimo Please take a final look!

**Review by ymorimo — APPROVED** [review]
LGTM!

**Pull request merged**